SPECS/perl/perl.spec

```diff
@@ -34,6 +35,7 @@ The Perl package contains the Practical Extraction and
 Report Language.
 %prep
 %setup -q
```

Comment:
Suggests changing to %autosetup -p1

Reply:
After speaking offline, I'm going to decline to change this.